File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        if (nums.length <= 1){
            return false
        }
        for(let n in nums){
            let t=nums.pop()
            if(nums.includes(t)) {
                return true
            }
        }

        return false
    }
}

File: Data Structures & Algorithms/evaluate-reverse-polish-notation/submission-0.java
class Solution {
    public int evalRPN(String[] tokens) {
        Deque<Integer> nums = new ArrayDeque<>();
        
        for (String token : tokens) {
            if (token.equals("+") || token.equals("-") || token.equals("*") || token.equals("/")) {
                int num2 = nums.pop();
                int num1 = nums.pop();
                
                switch (token) {
                    case "+": nums.push(num1 + num2); break;
                    case "-": nums.push(num1 - num2); break;
                    case "*": nums.push(num1 * num2); break;
                    case "/": nums.push(num1 / num2); break;
                }
            } else {
                nums.push(Integer.valueOf(token));
            }
        }
        
        return nums.pop();
    }
}